core/templates/dev/head/pages/exploration_player/FeedbackPopupDirective.js

```diff
@@ -55,6 +55,8 @@ oppia.directive('feedbackPopup', [
           // elements on the same page.
           $scope.feedbackPopoverId = (
             'feedbackPopover' + Math.random().toString(36).slice(2));
+          $scope.feedbackTitle = ('Feedback when the user was at card ' +
+            PlayerPositionService.getCurrentStateName());
```

Comment:
Perhaps quotation marks should surround the card name.

Also, when the contents of a parentheses span multiple lines, break after the first '('.

Reply:
Also, just to check, did you manually test with the longest possible state name? Just to make sure there are no validation errors (since there used to be truncation).

Reply:
thanks, @seanlip for pointing this, I have observed this and when the card name is of max. characters that are 50 then the feedback looks like this.
![Screenshot from 2019-04-18 19-25-50](https://user-images.githubusercontent.com/34139754/56366773-7d707980-6211-11e9-8df7-b9de4b6ffb21.png)

So to make it more user-friendly made some changes and the new view is like this
![Screenshot from 2019-04-18 19-37-52](https://user-images.githubusercontent.com/34139754/56366974-e48e2e00-6211-11e9-8a3b-e8dce26a59a3.png)
![Screenshot from 2019-04-18 19-37-58](https://user-images.githubusercontent.com/34139754/56366980-e6f08800-6211-11e9-81f0-ca8ff3e7039f.png)
![Screenshot from 2019-04-18 19-38-02](https://user-images.githubusercontent.com/34139754/56366990-e8ba4b80-6211-11e9-98dc-76a1e113cf39.png)

Reply:
PTAL, thanks